.cicd/helpers/file-hash.sh

```diff
@@ -4,4 +4,4 @@ set -eo pipefail
 FILE_NAME=$(basename $1 | awk '{split($0,a,/\.(d|s)/); print a[1] }')
 export DETERMINED_HASH=$(sha1sum $1 | awk '{ print $1 }')
 export HASHED_IMAGE_TAG="eos-${FILE_NAME}-${DETERMINED_HASH}"
-export FULL_TAG="eosio/ci:$HASHED_IMAGE_TAG"
\ No newline at end of file
+export FULL_TAG="${IMAGE_NAME:-"eosio/ci"}:$HASHED_IMAGE_TAG"
```

Comment:
This enable usage of a different identifier for the base image, so they can be stored in a different distribution system.

Reply:
👍